### PR TITLE
some minor fix

### PR DESCRIPTION
**Pull request by hero78119**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,8 @@
     "jsdom": "^3.0.0",
     "jshint": "^2.5.1",
     "mocha": "^2.0",
-    "react-i13n": "^0.1.8"
+    "react-i13n": "^2.0.4",
+    "debug": "~2.2.0"
```
Comment (kaesonho) [inline package.json]: debug should be in `dependencies`, can we keep that?

File: package.json [hunk 2]
```diff
@@ -50,9 +51,6 @@
   "directories": {
     "test": "tests"
   },
-  "dependencies": {
-    "debug": "~2.2.0"
```
Comment (kaesonho) [inline package.json:54]: one more feedback, please use `^2.2.0` to pick minor version bump.

Comment (hero78119) [inline package.json:54]: got

File: package.json (hunk 1 above)
Comment (hero78119) [inline package.json]: ok, just rollback!

File: package.json [hunk 3]
```diff
@@ -51,7 +51,8 @@
     "test": "tests"
   },
   "dependencies": {
-    "debug": "~2.2.0"
+    "debug": "^2.2.0",
+    "react-i13n": "^2.0.4"
```
Comment (kaesonho) [inline package.json:55]: <strike>`react-i13n` is only used on test, please move to `devDependencies`</strike>  NVM, I'm good with dependencies

**Comment by kaesonho** [comment]
:+1:

**Pull request merged**